roles/load_aware_deploy_trimaran/templates/trimaran.yaml.j2

```diff
@@ -23,17 +23,22 @@ stringData:
           - name: NodeResourcesBalancedAllocation
           - name: NodeResourcesLeastAllocated
           enabled:
-          - name: TargetLoadPacking
+          - name: {{ load_aware_deploy_trimaran_plugin | default('NA') }}
```

Comment:
I think you don't need `| default('NA')`, the variables will always have a value. Or fail earlier if they must have a value and the user didn't provide one

Reply:
hum, or you pass the defaults for the `undeploy` role, is that it? makes sense then

Reply:
not sure what's the proper way to go 🤔 🤔 🤔 
I don't like these `| default('NA')` in the template, they complicate the reading with unnecessary things.

🤔 
we need to find a way to pass all the mandatory variable in the `undeploy` function ...

Reply:
The `| default('NA')` definitely feels hacky. Let me see if I can think of something better that doesn't clog up the template.